content/derisking/index.html

```diff
@@ -0,0 +1,95 @@
+---
+title: De-risking government technology
+permalink: /derisking/
+layout: layouts/wide
+tags: derisking
+eleventyNavigation:
+  key: derisking-home-index
+  parent: derisiking-home
+  order: 1
+  title: Derisking Guide
+---
+
+<section class="usa-section--dark hero padding-y-6">
+  <div class="grid-container">
+    <div class="grid-row grid-gap flex-align-center">
```

Comment:
Just observing that the responsive behavior on mobile is different from the original, but I do like this one better!